src/codegen/clang.rs

```diff
@@ -0,0 +1,8 @@
+use anyhow::{bail, Result};
+
+pub fn get_path() -> Result<String> {
+    match std::env::var("CLANG") {
+        Ok(v) => Ok(v),
```

Comment:
Should we check that the path exists and is a file?

Reply:
I do not think so? Since `Command` can immediately return an error if it does not exist or is not a file, that would not be the responsibility of `clang_path`. Instead, we can implement `validate_path`, but I think it is better to leave it to Command for now. What do you think?

Reply:
Sounds good to me! I'm just curious, but would you mind linking me to the clap docs so I can read about where this happens?

Reply:
Sure, `Command` is defined under `std::process`, https://doc.rust-lang.org/std/process/struct.Command.html